vividus-to-azure-devops-exporter/src/main/java/org/vividus/azure/devops/exporter/AzureDevOpsExporter.java

```diff
@@ -46,40 +52,60 @@ public AzureDevOpsExporter(AzureDevOpsExporterOptions options, AzureDevOpsFacade
 
     public void exportResults() throws IOException
     {
+        Map<Integer, Scenario> exportedScenarios = new HashMap<>();
+
         for (Story story : OutputReader.readStoriesFromJsons(options.getJsonResultsDirectory()))
         {
             LOGGER.atInfo().addArgument(story::getPath).log("Exporting scenarios from {} story");
 
-            story.getFoldedScenarios().forEach(scenario -> exportScenario(story.getPath(), scenario));
+            story.getFoldedScenarios().stream()
+                                      .map(scenario -> exportScenario(story.getPath(), scenario))
+                                      .flatMap(Optional::stream)
+                                      .forEach(entry -> exportedScenarios.put(entry.getKey(), entry.getValue()));
+        }
+
+        if (exportedScenarios.isEmpty())
+        {
+            LOGGER.atInfo().log("No scenarios were exported");
```

Comment:
when can it happen?

Reply:
when all scenario were skipped or failed to export (IOException, NotUniqueMetaValueException, SyntaxException errors)